admin/src/main/resources/io/buoyant/admin/js/main-namerd.js

```diff
@@ -23,7 +23,12 @@ require.config({
 require([
   'jQuery',
   'bootstrap',
-  'src/dashboard_delegate'
-], function ($, bootstrap, namerdDtabPlayground) {
-  new namerdDtabPlayground();
+  'src/dashboard_delegate',
+  'src/logging'
+], function ($, bootstrap, namerdDtabPlayground, loggingConfig) {
+  if(window.location.pathname.indexOf("/dtab") >= 0){
```

Comment:
this is how you do "contains" in javascript???

Reply:
sadly yes. In JS's defense, this was how it was done pre-ES6, now you can use `.contains` but that is not supported in all browser versions.